src/create-sync.coffee

```diff
@@ -0,0 +1,94 @@
+_ = require 'underscore'
+Backbone = require 'backbone'
+Backbone.$ = require 'jquery' # TODO remove after upgrading to backbone 1.2+
+
+Utils = require './utils'
+
+
+module.exports = (hostOptions) ->
```

Comment:
Velleity: Documentation on the usages of `hostOptions`?

Reply:
And the types. e.g. `withCredentials: Boolean`?

Reply:
Good call, we'll add some